numpy/core/tests/test_umath.py

```diff
@@ -1339,6 +1339,12 @@ def test_reduce_warns(self):
                     assert_equal(np.min(r), np.nan)
                 assert_equal(len(sup.log), n)
 
+    def test_minimize_warns(self):
+        # gh 11589
+        with suppress_warnings() as sup:
+            sup.record(RuntimeWarning)
+            np.minimum(np.nan, 1)
+
```

Comment:
`assert_warns` should be fine here? But the test does not actually check that the one warning is recorded like this.